prow/jobs/jinja/presubmits/service_tests.jinja2

```diff
@@ -88,7 +88,9 @@
     spec:
       serviceAccountName: pre-submit-service-account
       containers:
-      - image: public.ecr.aws/m5q3e4b2/prow:prow-integration-0.0.5
```

Comment:
@vijtrip2 @RedbackThomson so does this mean we can delete the prow-integration image from our internal ack-infra ECR Public repos and this image definition file? https://github.com/aws-controllers-k8s/test-infra/blob/main/prow/jobs/images/Dockerfile.integration